tower-batch-control/Cargo.toml

```diff
@@ -27,8 +29,8 @@ rand = { version = "0.8.5", package = "rand" }
 
 tokio = { version = "1.28.2", features = ["full", "tracing", "test-util"] }
 tokio-test = "0.4.2"
-tower-fallback = { path = "../tower-fallback/" }
+tower-fallback = { path = "../tower-fallback/", version = "0.2.40" }
 tower-test = "0.4.0"
 
-zebra-consensus = { path = "../zebra-consensus/" }
-zebra-test = { path = "../zebra-test/" }
+zebra-consensus = { path = "../zebra-consensus/", version = "1.0.0-beta.25" }
```

Comment:
> We need to fix a cyclic dependency between `tower-batch-control` and `zebra-consensus` to publish

The supported workaround is "remove the `version` dependency".

```suggestion
# We can't specify a version here, because that would create a circular dependency during crates.io publishing:
# <https://github.com/rust-lang/cargo/pull/7333>
#
# TODO
# place these tests behind a feature that's off by default, but activate that feature in CI,
# so the other tower-batch-control tests can be run when the crate is downloaded from crates.io
zebra-consensus = { path = "../zebra-consensus/" }
```

Reply:
You're right, this works

Reply:
re: 3b06b054